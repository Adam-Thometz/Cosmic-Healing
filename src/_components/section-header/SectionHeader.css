.SectionHeader {
  font-family: 'Nimbus Roman';
  margin: 16px 0;
  margin: 0 0 24px 0;
}

h2 {
  font-size: 36px;
  line-height: 54px;
}

h3 {
  font-size: 24px;
  line-height: 32px;
}