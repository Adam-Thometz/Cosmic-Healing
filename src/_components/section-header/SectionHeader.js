import React from 'react';

import './SectionHeader.css';

const SectionHeader = ({ text, small = false }) => {
  
  return small ? (
    <h3 className="SectionHeader">{text}</h3>
  ) : (
    <h2 className="SectionHeader">{text}</h2>
  )
};

export default SectionHeader;