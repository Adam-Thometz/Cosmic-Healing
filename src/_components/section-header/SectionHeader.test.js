import React from "react";
import { render } from "@testing-library/react";
import SectionHeader from './SectionHeader';

describe('SectionHeader component', () => {
  it('renders without crashing', () => {
    render(<SectionHeader />);
  });

  it('matches the snapshot', () => {
    const { asFragment } = render(<SectionHeader />);
    expect(asFragment()).toMatchSnapshot();
  });
});