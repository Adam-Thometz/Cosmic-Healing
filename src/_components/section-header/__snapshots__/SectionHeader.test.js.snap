// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHeader component matches the snapshot 1`] = `
<DocumentFragment>
  <h2
    class="SectionHeader"
  />
</DocumentFragment>
`;
